etc/di.xml

```diff
@@ -343,4 +343,35 @@
     <type name="Magento\Config\Model\Config">
         <plugin name="boltRequestAfterConfigurationChange" type="Bolt\Boltpay\Plugin\Magento\Config\Model\ConfigPlugin" />
     </type>
+    <type name="Bolt\Boltpay\Model\CatalogIngestion\ProductEventManager">
+        <arguments>
+            <argument name="logger" xsi:type="object">Bolt\Boltpay\Logger\CatalogIngestionLogger</argument>
+        </arguments>
+    </type>
+    <type name="Bolt\Boltpay\Model\CatalogIngestion\StoreConfigurationManager">
+        <arguments>
+            <argument name="logger" xsi:type="object">Bolt\Boltpay\Logger\CatalogIngestionLogger</argument>
+        </arguments>
+    </type>
+    <type name="Bolt\Boltpay\Model\CatalogIngestion\ProductEventProcessor">
+        <arguments>
+            <argument name="logger" xsi:type="object">Bolt\Boltpay\Logger\CatalogIngestionLogger</argument>
+        </arguments>
+    </type>
+    <virtualType name="Bolt\Boltpay\Logger\CatalogIngestionLogger"
+                 type="Bolt\Boltpay\Logger\Logger">
+        <arguments>
+            <argument name="name" xsi:type="string">BoltProductEvents</argument>
+            <argument name="handlers"  xsi:type="array">
+                <item name="system" xsi:type="object">Bolt\Boltpay\Logger\CatalogIngestionHandler</item>
+            </argument>
+        </arguments>
+    </virtualType>
+    <virtualType name="Bolt\Boltpay\Logger\CatalogIngestionHandler"
+                 type="Bolt\Boltpay\Logger\Handler">
+        <arguments>
+            <argument name="fileName" xsi:type="string">/var/log/bolt-catalog-ingestion.log</argument>
```

Comment:
What happens if php doesn't have rights for this path?

Reply:
the same as on default magento behaviour, there is no difference. 
https://github.com/Seldaek/monolog/blob/0c375495d40df0207e5833dca333f963b171ff43/src/Monolog/Handler/StreamHandler.php#L116